config/default.yml

```diff
@@ -405,6 +405,11 @@ Rails/SafeNavigation:
   # implement the intended method. `try` will not raise an exception for this.
   ConvertTry: false
 
+Rails/SafeNavigationWithBlank:
+  Description: 'Avoid `foo&.blank?` in conditionals.'
+  Enabled: true
```

Comment:
I think this needs to be marked unsafe. And it may be useful to add the following example as a comment.

```diff
 Enabled: true
+# While the safe navigation operator is generally a good idea, when
+# checking `foo&.blank?` in a conditional, `foo` being `nil` will actually
+# do the opposite of what the author intends.
+#
+# foo&.blank? #=> nil
+# foo.blank? #=> true
+SafeAutoCorrect: false
 VersionAdded: '2.4'
```

Reply:
Ah, I forgot to comment 💦 I think this cop can have autocorrect implemented.

Reply:
Implemented in 5ee1d23f02b74cdffea0d7fe1eae866eb7d0fc4b.